Source: Peter Volkov <pva@gentoo.org>
Upstream: notified (sent email to <adrian2 AT caribe DOT net>
Reason: bugs.gentoo.org/237828 comment #1
With md5sum-external fails on filenames with spaces in their names.

--- fdupes.c	2008-09-17 08:33:26 +0000
+++ fdupes.c	2008-09-17 08:33:42 +0000
@@ -301,7 +301,7 @@
     exit(1);
   }
 
-  sprintf(command, "%s %s", EXTERNAL_MD5, filename);
+  sprintf(command, "%s '%s'", EXTERNAL_MD5, filename);
 
   result = popen(command, "r");
   if (result == NULL) {

